terraform/graph_builder_apply.go

```diff
@@ -168,20 +168,10 @@ func (b *ApplyGraphBuilder) Steps() []GraphTransformer {
 			Config:  b.Config,
 			State:   b.State,
 			Schemas: b.Schemas,
-			Destroy: b.Destroy,
 		},
 
-		// Handle destroy time transformations for output and local values.
-		// Reverse the edges from outputs and locals, so that
-		// interpolations don't fail during destroy.
 		// Create a destroy node for outputs to remove them from the state.
-		GraphTransformIf(
-			func() bool { return b.Destroy },
-			GraphTransformMulti(
-				&DestroyValueReferenceTransformer{},
-				&DestroyOutputTransformer{},
-			),
-		),
+		&DestroyOutputTransformer{Destroy: b.Destroy},
```

Comment:
Oh, I guess _this_ is still a difference between `terraform plan -destroy` and `terraform destroy`. _One day...._